.github/workflows/package-checks.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ['3.10', '3.11']
```

Comment:
Should 3.12 be in that list?

(`test_with_missing_lzma_runtime` is failing)

Reply:
Good call. Will add

Reply:
Looks like `conda-verify` doesn't support Python 3.12 yet so we can't add this yet